<?xml version="1.0" encoding="UTF-8" ?>
<testsuites errors="1"
            failures="0"
            tests="1">

  <testsuite name="wantedQueriesTest"
    errors="1" failures="0" hostname="" id="" package="wantedQueriesTest" skipped="0"
    tests="1" time="0.01100" timestamp="Mon, 17 Dec 2018 22:12:33 GMT">
  
    <testcase name="Submit A Valid" classname="wantedQueriesTest" time="0.01100" assertions="0">
    </testcase>
  

  
    <system-err>
        Error while running [Wanted Queries Test / Submit A Valid]:

  ReferenceError: test is not defined
    at Object.Submit A Valid (C:\Users\Bracken\Desktop\DevMtn\wanted-queries\tests\wantedQueriesTest.js:17:34)
    at Module.call (C:\Users\Bracken\AppData\Roaming\npm\node_modules\nightwatch\lib\runner\module.js:62:34)
    at C:\Users\Bracken\AppData\Roaming\npm\node_modules\nightwatch\lib\runner\testcase.js:70:29
    at _fulfilled (C:\Users\Bracken\AppData\Roaming\npm\node_modules\nightwatch\node_modules\q\q.js:834:54)
    at C:\Users\Bracken\AppData\Roaming\npm\node_modules\nightwatch\node_modules\q\q.js:863:30
    at Promise.promise.promiseDispatch (C:\Users\Bracken\AppData\Roaming\npm\node_modules\nightwatch\node_modules\q\q.js:796:13)
    at C:\Users\Bracken\AppData\Roaming\npm\node_modules\nightwatch\node_modules\q\q.js:556:49
    at runSingle (C:\Users\Bracken\AppData\Roaming\npm\node_modules\nightwatch\node_modules\q\q.js:137:13)
    at flush (C:\Users\Bracken\AppData\Roaming\npm\node_modules\nightwatch\node_modules\q\q.js:125:13)
    at process._tickCallback (internal/process/next_tick.js:61:11)
    </system-err>
  

  
  </testsuite>
</testsuites>
